apps/financial-aid/web-veita/graphql/sharedGql.ts

```diff
@@ -101,7 +101,24 @@ export const GetMunicipalityQuery = gql`
     municipality(input: $input) {
       id
       name
-      settings
+      settings {
+        aid {
+          ownApartmentOrLease
+          withOthersOrUnknow
+          withParents
+        }
+      }
+    }
+  }
+`
+
+export const GetMunicipalityIdQuery = gql`
+  query GetMunicipalityIdQuery($input: MunicipalityQueryInput!) {
+    municipality(input: $input) {
+      id
+      settings {
+        homePage
+      }
```

Comment:
Thie query is called `GetMunacipalityIdQuery` but `id` is an input for the query, doesn't it make more sense to call the query `GetMunacipalityHomePageQuery` ? And should we  skip getting the id and only get the homepage?